.circleci/config.yml

```diff
@@ -1,5 +1,47 @@
 version: 2
 jobs:
+  build-test-linux:
+    docker:
+      - image: matteblair/docker-tangram-linux:latest
+    steps:
+      - checkout
+      - run: git submodule update --init
+      - run: make cmake-linux CMAKE_OPTIONS="-DTANGRAM_USE_SYSTEM_FONT_LIBS=1 -DTANGRAM_USE_SYSTEM_GLFW_LIBS=1 -DUNIT_TESTS=1 -DBENCHMARK=1 -GNinja"
+      - run: ninja -C build/linux -j 4
+      - run:
+          name: Run unit tests
+          command: |
+            cd ./build/linux/bin
+            for file in unit/*.out
+              do
+                echo "Running ${file}"
+                $file
+              done
+      - run:
+          name: Run benchmarks
+          command: |
+            # Use a stored tile for testing
+            cp bench/test_tile_10_301_384.mvt build/linux/bin/tile.mvt
+            cd ./build/linux/bin
+            for file in bench/*.out
+              do
+                echo "Running ${file}"
+                $file
+              done
+  build-android:
+    docker:
+      - image: lakoo/android-ndk:25-26.0.1-r15c
+    steps:
+      - checkout
+      - run: git submodule update --init
+      - run: cd platforms/android && ./gradlew demo:assembleDebug -Pandroid.injected.build.abi=armeabi-v7a
+  build-deploy-android-snapshot:
+    docker:
+      - image: lakoo/android-ndk:25-26.0.1-r15c
+    steps:
+      - checkout
+      - run: git submodule update --init
+      - run: cd platforms/android && ./gradlew uploadArchives
   build-ios:
     macos:
       xcode: "9.0"
```

Comment:
https://circleci.com/docs/2.0/testing-ios/#supported-xcode-versions

Reply:
CircleCI does support newer versions, but in general upgrading tools causes sadness. Unless there's a compelling reason to upgrade I'd prefer to leave it the same.

Reply:
Hmm, I agree with sadness!
But I am of an opinion that CI builds should reflect the build system we use for our local builds. At least the release builds should be done using the latest supported versions and if not we should document the releases to be build on a specific xcode version like in this case.

Reply:
Updated to Xcode 9.2.0, which is what I've got locally. Seems fine!